src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -356,12 +356,14 @@ def create_user(client, user_principal_name, display_name, password,  # pylint:
 
 
 def list_groups(client, display_name=None, query_filter=None):
+    '''
```

Comment:
Let me leave as it is. I invoked the pep8 style check, the code is fine. Of course, we can include new rules for this and update all code at one shot